### DISCUSSION TITLE
ScriptReference/FrameTimingManager

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#154] [Notes]
# ScriptReference/FrameTimingManager 3c9bdc1b

https://docs.unity3d.com/ScriptReference/FrameTimingManager.html

**Comment by vertxxyz** [comment]
See [FrameTimingManager](https://docs.unity3d.com/2022.2/Documentation/Manual/frame-timing-manager.html) Manual documentation (2022.2+) for a detailed rundown.
There is also an accompanying [blog post](https://blog.unity.com/technology/detecting-performance-bottlenecks-with-unity-frame-timing-manager).
